cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -442,9 +442,9 @@
   /**
    * <code>execution.task.strategy</code>
    */
-  public static final String EXECUTION_TASK_STRATEGY_CONFIG = "execution.task.strategy";
-  private static final String EXECUTION_TASK_STRATEGY_DOC = "The strategy used to determine execution order for generated partition"
-      + "movement tasks.";
+  public static final String REPLICA_MOVEMENT_STRATEGIES_CONFIG = "replica.movement.strategies";
```

Comment:
Can we add a sanity check in this class to ensure that the list received from `REPLICA_MOVEMENT_STRATEGIES_CONFIG` is not empty? -- i.e. similar to the sanity check we have for `DEFAULT_GOALS_CONFIG`.